.github/workflows/post-release.yml

```
@@ -66,7 +66,7 @@ jobs:
         run: cp .golangci.next.reference.yml .golangci.reference.yml
 
       - name: Update JSON schema files
-        run: cp docs/static/schema/next.jsonschema.json docs/static/schema/jsonschema.json
+        run: cp .schema/next.jsonschema.json .schema/jsonschema.json
```

If this is current workdir, I think `./` is more common and less confusing unless it's a hidden folder in cwd.

```suggestion
        run: cp ./schema/next.jsonschema.json ./schema/jsonschema.json
```

---

I changed to `jsonschema`